packages/block-editor/src/components/responsive-block-control/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Any idea how this snapshot ended up minified? Makes it impossible to review changes to the component 😞

Reply:
@tellthemachines It's quite likely that because the test is [using a _string_ of HTML](https://github.com/WordPress/gutenberg/blob/11eaaee2e3cfdb62c40b7d131c7bd021da1fca99/packages/block-editor/src/components/responsive-block-control/test/index.js#L121) for the snapshot, it doesn't know how to make sense of the string as markup of elements to break across lines, vs. say a React element tree. I'd agree though that it sort of defeats the purpose of a snapshot if a future maintainer is unable to assess whether changes in the snapshot are sensible.

I'm not sure exactly what our current configuration will support for those snapshots, but I'd wonder if it could be using the DOM element or if we could adapt somehow for the React element to be used.